### PR TITLE
Add hold step on CCP

### PR DESCRIPTION
**Pull request by evrardjp**
This adds the way to "hold" instances when a failure happens, for debugging purposes.

### CODE REVIEW

File: jenkins/ci.suse.de/ccp.yaml [hunk 1]
```diff
@@ -29,6 +29,11 @@
           default: master
           description: The socok8s branch
 
+      - string:
```
Comment (stefannica) [inline jenkins/ci.suse.de/ccp.yaml]: You can model this as a bool parameter, which will show up as a checkbox in the UI, e.g.:

```
      - bool:
          name: want_caasp
          default: false
          description: >-
            Deploy CaaSP using the caasp-openstack-heat-templates.
```

Comment (JanZerebecki) [reply jenkins/ci.suse.de/ccp.yaml]: A true bool is represented as the string "true", see https://github.com/SUSE-Cloud/automation/blob/c0b6a344d0a8edde7fd5e9e07050f5b63b558458/jenkins/ci.suse.de/cloud-mkphyscloud.yaml#L111

Comment (evrardjp) [reply jenkins/ci.suse.de/ccp.yaml]: Thanks, will update.

**Comment by evrardjp** [comment]
Seems to work. Waiting for reviews now.

**Pull request merged**